src/main/java/Nic.java

```diff
@@ -0,0 +1,52 @@
+
+public class Nic {
+
+
+    public static void main(String[] args) {
```

Comment:
Try to avoid lengthy methods. Keep each method length to about 30 lines of code.